concepticondata/references/references.bib

```diff
@@ -2981,6 +2981,19 @@ @article{Alario1999
   publisher={Springer}
 }
 
+@incollection{Saxena2013,
+    address = {Berlin},
+    author = {Saxena, Anju and Borin, Lars},
+    booktitle = {Approaches to Measuring Linguistic Differences},
+    isbn = {978-3-11-030525-8},
+    keywords = {irreproducible research},
```

Comment:
This keyword might not be relevant for everybody or maybe misleading.

Reply:
I'll delete it then.